orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -115,7 +115,8 @@ def load_features(self):
         self.selected_features = []
         if self.corpus is not None:
             self.features = [(meta.name, vartype(meta)) for meta in self.corpus.domain.metas] + \
-                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes]
+                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes] + \
+                            [(class_var.name, vartype(class_var)) for class_var in self.corpus.domain.class_vars]
```

Comment:
Since we don't use `iter(domain)` a lot, we can discuss it (on Friday?) and change it.

Most algorithms iterated just over `variables` (= `attributes + [class_var]`). If this is still the case, it would be an argument for keeping `Domain.__iter__` as it is -- it's more useful this way.